vscode-lean4/src/abbreviation/abbreviations.json

```diff
@@ -14,6 +14,8 @@
     "[--]": "⁅$CURSOR⁆",
     "nnnorm": "‖$CURSOR‖₊",
     "norm": "‖$CURSOR‖",
+    "floor": "⌊$CURSOR⌋",
+    "ceil": "⌈$CURSOR⌉",
```

Comment:
Should we have `nfloor` and `nceil` for the `₊` versions (`Nat.floor` and `Nat.ceil`)?

Reply:
That is a good idea.